starter-code/index.html

```diff
@@ -6,9 +6,69 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
   <link rel="stylesheet" href="./css/style.css">
-  <script type="text/javascript" src="./js/index.js"></script>
   <title>Ironhack cart</title>
 </head>
 <body>
+
+<header>
+  <h1>IronHack Merchandising Shop</h1>
+</header>
+
+<div class="products">
+
+  <!------------------------------------------------------------------------->
+  <div class="product">
+
+    <span class="product-name">IronBubble-head</span>
+    <span class="price-one-unit money">25.00</span>
+    <span>
+      <label class="quantity">QTY</label>
+      <input class='qty-input' type="number" step="1" placeholder="0">
+    </span>
+    <span class="total-price money">0.00</span>
+    <span><button class="btn btn-delete">Delete</button></span>
```

Comment:
You don't really need to put the button inside a span here :)

Reply:
👍